CMakeLists.txt

```diff
@@ -19,6 +19,9 @@ endif()
 set(OMIM_ROOT ${CMAKE_SOURCE_DIR})
 set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} "${OMIM_ROOT}/cmake")
 
+set(CMAKE_POSITION_INDEPENDENT_CODE ON)
+set(EXECUTABLE_OUTPUT_PATH ${CMAKE_CURRENT_BINARY_DIR})
+
```

Comment:
How about this?
```cmake
set(LIBRARY_OUTPUT_PATH ${CMAKE_CURRENT_BINARY_DIR})
```
It was also removed
If this var really works are we OK not to have it?